auto3dseg/algorithm_templates/segresnet2d/scripts/algo.py

```diff
@@ -202,7 +203,7 @@ def fill_template_config(self, data_stats_file, output_path, **kwargs):
             )
 
         # customize parameters for gpu
-        if kwargs.pop("gpu_customization", False):
+        if kwargs.pop("gpu_customization", True):
```

Comment:
@dongyang0122 I lost track when reading the code. I think we discussed this and agreed we want it default to be `False`. Right?

Reply:
right. update. thanks!